### ISSUE TITLE
Different theme settings for filenames and other columns in List layout

### ISSUE THREAD

**Issue by vladstudio** [mikekazakov/nimble-commander#118]
It's a good idea to make Extension, Size, etc less prominent than Name column. Compare with how Finder (on the right) does that:

<img width="1442" alt="image" src="https://github.com/mikekazakov/nimble-commander/assets/914320/deaaf455-282a-4ea5-a12f-69415c07736c">

Only the first column uses black text.
Ideally, I'd be able to configure text color separately for Names column and all other columns. But then I'd have to provide values for default state, selected, active, hidden file. Hmm!

**Comment by mikekazakov** [comment]
Fascinating, I never realized Finder dims the font color for other columns. Has it always been like that?
Perhaps this can be set for all colors, e.g., as an alpha value set for all non-primary columns.
Would, say, setting the alpha to 80% work? Customizable, of course.

**Comment by tiagof** [comment]
@mikekazakov are you open to a PR for this feature that allows customising the opacity of the "other columns" based on a theme parameter?

**Comment by mikekazakov** [comment]
Sure, PR is welcome!
Though I think it's necessary to first check how this would work with different themes and different coloring rules...

**Comment by tiagof** [comment]
Well, the idea is to "simply" apply the opacity value to the colors, whatever they are. 
If a specific alpha value does not work well with the current theme colors, it's the user's responsibility to adjust and come up with a "good" theme.

**Comment by mikekazakov** [comment]
Yes, that's understood. The question is mostly not about the "default" filename colors, but more complex interactions - e.g. a selected and focused row - how would it work then overall.  
Could you provide any screenshots showing how it looks?

**Comment by tiagof** [comment]
I would but for some reason my xcode stopped building the project. No code changes, this is directly from a main branch checkout.

As soon as I understand what's happening I'll provide them

<img width="1260" alt="Image" src="https://github.com/user-attachments/assets/4d08f7ec-339a-4abf-a72d-276d235645ca" />

**Comment by mikekazakov** [comment]
I hazard to guess that Xcode was automatically updated to v16.3?  
There are some incompatibilities within the source code, I'll push a fix this evening.

**Comment by tiagof** [comment]
> I hazard to guess that Xcode was automatically updated to v16.3? There are some incompatibilities within the source code, I'll push a fix this evening.

spot on!

...damn it!

**Comment by mikekazakov** [comment]
That's why I prefer to download and install it manually, instead of pulling from Mac App Store - less surprises like this...

**Comment by tiagof** [comment]
Here they are!

#### Light theme with 50%, 70% and 80% opacity

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/4085fdf0-099a-4c41-919d-5b9fd95d5b5b" />

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/dd186dea-c30a-46f1-b188-e8a1ed919117" />

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/90d42c3d-de45-4fbc-89dc-4645be910bf2" />

#### Dark theme with 50%, 70% and 80% opacity

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/3d75d4c6-a6be-46f0-97cd-92a512c618d0" />

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/ca31162d-ec1f-4854-8963-8bdc679c3870" />

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/9164f7a6-4ee0-4cb2-8199-2ad0250ffcac" />

#### Safari Light theme with 70% opacity

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/b8ab2e92-1127-489e-904c-fc891fac73ab" />

#### Safari Dark theme with 70% opacity

<img width="1166" alt="Image" src="https://github.com/user-attachments/assets/02ee4c27-819b-4c00-b8f1-dfa1d830c44a" />

**Comment by mikekazakov** [comment]
Thank you for the screenshots.  
I think this approach can work as a practical compromise and up to one's taste in such subtleties.

In the perfect world I guess these colors should be specified manually to fully preserve legibility.  
i.e. expanding the coloring rules to carry 4 colors per rule: (regular, focused) x (primary, secondary), but the total number of these colors would get out of control very quickly: e.g. 12 for the Light theme and 20 for the Classic theme - that's likely beyond reason.

**Comment by tiagof** [comment]
I thought about that too but then it would be as you say: too many colors to configure.
Also, using the opacity approach, you adjust 1 setting that impacts "all" rules.

I'll go ahead and open a PR for this.
Question: what would you call this setting? 
Currently I have this but I don't like the name...

<img width="842" alt="Image" src="https://github.com/user-attachments/assets/e1d43ecc-0e13-480d-bcdd-52902faf4325" />

**Comment by mikekazakov** [comment]
Tricky... maybe "Secondary text opacity"?

**Comment by mikekazakov** [comment]
Fixed with #554, thanks @tiagof!

**Comment by mikekazakov** [comment]
> Hey guys, where can I download some themes?

You can find a few themes here in Discussions on GitHub, but that's about it.
